sass/components/_post.scss

```diff
@@ -2038,7 +2039,6 @@
 
     // MM-38006 - There's a bug on Chrome/Safari where a collapsed post will sometimes have its contents scroll. This
     // is prevented by using clip instead of hidden, but that's not supported on the stable version of Safari yet.
-    overflow: hidden;
```

Comment:
have we verified MM-38006 is not an issue anymore?

Reply:
I am not sure about this particular ticket, but there was two overflow properties specified on this selector, so one was not being applied. I have left the one that was applied. Seems that this wasn't fixing the bug in the first place 🤷

Reply:
Looks like Safari's still the odd one out here since it still doesn't support `overflow: clip`

https://caniuse.com/mdn-css_properties_overflow_clip

The reason we applied both of those is that only the last valid one has an effect so a browser that supports `overflow: clip` uses that and one that doesn't uses `overflow: hidden`

Reply:
I can add it back for safari, but in the way it was applied, on normal browsers was also using `overflow: hidden`.

Reply:
Yeah, could you please change it back? This doesn't sound like the exact bug we had before, but it certainly has some weird behaviour going on on Safari 15.6 when you hover a post which doesn't repro on Firefox or Safari 16.1
![Screen Shot 2023-01-05 at 3 59 51 PM](https://user-images.githubusercontent.com/3277310/210879108-ffc4dddb-2a10-45cd-a569-019c43a652c1.png)

And I think you might have that backwards. Firefox, at least, uses `clip` there
![Screen Shot 2023-01-05 at 3 56 09 PM](https://user-images.githubusercontent.com/3277310/210879387-82debb3a-eceb-4972-acd8-b9c7984d12b0.png)